src/main/java/nl/novi/techiteasycontroller/Services/RemoteControllerService.java

```diff
@@ -0,0 +1,92 @@
+package nl.novi.techiteasycontroller.Services;
+
+import nl.novi.techiteasycontroller.Dto.Input.RemoteControllerInputDto;
+import nl.novi.techiteasycontroller.Exceptions.RecordNotFoundException;
+import nl.novi.techiteasycontroller.Model.RemoteController;
+import nl.novi.techiteasycontroller.Repository.RemoteControllerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+
+@Service
+public class RemoteControllerService  {
+
+    @Autowired
```

Comment:
Hier mag en kan je @Autowired gebruiken, maar je gebruikt overal "constructor injection", dan zou ik dat hier ook doen in het kader van consistentie